jenkins/setup.sh

```diff
@@ -21,10 +21,11 @@ fi
 
 logdir=/tmp/gridtools/ # log to subfolder to workaround https://webrt.cscs.ch/Ticket/Display.html?id=38406
 mkdir -p $logdir
+chmod +rwx $logdir
```

Comment:
Not sure if it's a problem. What will happen if the directory already existed but is now executed by a different user?

Reply:
You mean if the directory is owned by somebody else? Then the command will probably fail, but this should not matter as long as the directory has the correct permissions set already. Though at some point it would be good if we had better error handling, then it could matter indeed :P

Reply:
I was just not sure if we stop on error.